### DISCUSSION TITLE
Move users from one channel to another

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Lern/Community#104] [Query] [answered]
Hi team
Hope you are doing well!
We are using multiple channels for different functionalities. we have one scenario where we want to move few users from one channel to another.
eg. user abc is of channel X and we want to move it to Channel Y.

We tried user update API but couldn't work.  Also we are not sure about user migration steps. 
This seems to be only related to custodian Org. How we can do this? also after changing channel will users content affects??
Please help us to change channel Id of users 

@reshmi-nair , @BharathwajShankar ,

**Comment by reshmi-nair** [comment]
@paritshivani Migration api is used to move user from custodian or default tenant to other tenant organisations. Tenant to tenant movement is not available now.

**Comment by paritshivani** [comment]
@reshmi-nair , Is there any other way we can use same users in other tenant? We have one urgent requirement of this.

**Reply by reshmi-nair** [reply]
@paritshivani Seems like in migration api request, "forceMigration" flag can be passed to do this. I will confirm on this.

**Reply by reshmi-nair** [reply, marked as answer]
@paritshivani This api request can be used to do migration from one tenant to another
curl --location --request PATCH 'localhost:9000/private/user/v1/migrate' \
--header 'Content-Type: application/json' \
--data '{
    "request": {
        "userId": "ae36d90f-7e29-4f10-8b27-3fe8d0472fe1",
        "rootOrgId": "0137787306921246720",
        "forceMigration": true
    }
}'
